### ISSUE TITLE
groups in cross validation?

### ISSUE THREAD

**Issue by crsegerie** [AutoViML/Auto_ViML#22]
I would like to use the group params in the cross validation like in sklearn ? Is it possible to do so ?

Otherwise, thank you for your package, this is a very cool work here, especially the visualization part.

**Comment by rsesha** [comment]
Hi 👍
Thanks for the appreciation.
Is this what you are looking for?
[Link](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_validate.html)
groups: array-like of shape (n_samples,), default=None
Group labels for the samples used while splitting the dataset into train/test set. Only used in conjunction with a “Group” cv instance (e.g., GroupKFold).

Since I don't use GroupKFold, but just regular KFold, do you want an input argument that says "groups" and if you send the groups, I can use it with the GroupKFold?

Please let me know and I'll consider it.
Thanks
Ram

**Comment by crsegerie** [comment]
yes, this is exactly it!

**Comment by AutoViML** [comment]
Hi @crsegerie 👍 
Can you create a pull request and take a crack at writing some code? I don't have much time and would appreciate the help,
Thanks
AutoVimal
